db/migrate/20200521230648_remove_project_source.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveProjectSource < ActiveRecord::Migration
+  def change
+  	 Rails.cache.write('check:migrate:remove_project_source', Time.now)
```

Comment:
Please fix indentation

Reply:
Done

Reply:
Sawy, will there be another migration to delete the actual `project_source` database tables and columns? I can't see that anywhere.

Reply:
Oh I forgot to add the delete for `project_source` table will add to existing migration